jobs/JREGIONAL_RUN_MET_PB2NC_OBS_NDAS

```diff
@@ -76,7 +76,7 @@ NDAS observations.
 #
 #-----------------------------------------------------------------------
 #
-$SCRIPTSdir/exregional_run_met_pb2nc_obs.sh || \
+$SCRIPTSdir/exregional_run_met_pb2nc_obs_ndas.sh || \
```

Comment:
Why are we renaming all the pb2nc references to include references to NDAS? Is there not a potential future where we want to process other prepbufr files with the same logic?

Reply:
I only recently found out that prepbufr files come from other sources as well (e.g. GDAS).  I'll undo this and the name change to the j-job.